pkg/util/testingjobs/mpijob/wrappers_mpijob.go

```diff
@@ -195,6 +196,20 @@ func (j *MPIJobWrapper) UID(uid string) *MPIJobWrapper {
 	return j
 }
 
+// OwnerReference adds a ownerReference to the default container.
+func (j *MPIJobWrapper) OwnerReference(ownerName string, ownerGVK schema.GroupVersionKind) *MPIJobWrapper {
+	j.ObjectMeta.OwnerReferences = []metav1.OwnerReference{
```

Comment:
```suggestion
	j.ObjectMeta.OwnerReferences = append(j.ObjectMeta.OwnerReferences, metav1.OwnerReference{
```

Should we append owerReferences instead of overriding so that we can avoid caring if existing objects have owner?